sdk/monitor/azure-monitor-opentelemetry/pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[tool.azure-sdk-build]
+type_check_samples = false
+verifytypes = false
+pyright = false
+mypy = false
```

Comment:
Turning mypy off as suggested by @pvaneck. Not sure why it thinks these modules don't exist:
```
azure/monitor/opentelemetry/autoinstrumentation/_distro.py:20: error: Module "azure.monitor.opentelemetry._vendor.v0_39b0.opentelemetry.instrumentation.distro" has no attribute "BaseDistro"  [attr-defined]
azure/monitor/opentelemetry/_configure.py:35: error: Module "azure.monitor.opentelemetry._vendor.v0_39b0.opentelemetry.instrumentation.instrumentor" has no attribute "BaseInstrumentor"  [attr-defined]
azure/monitor/opentelemetry/_configure.py:110: error: "TracerProvider" has no attribute "add_span_processor"  [attr-defined]
azure/monitor/opentelemetry/_configure.py:126: error: "LoggerProvider" has no attribute "add_log_record_processor"  [attr-defined]
azure/monitor/opentelemetry/_configure.py:150: error: Unsupported right operand type for in ("Union[str, bool, int, float, Resource, Sequence[str], Sequence[bool], Sequence[int], Sequence[float], Sequence[MetricReader], Sequence[View]]")  [operator]
Found 5 errors in 2 files (checked 69 source files)
```
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2918486&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=e3f8f308-6e5f-58aa-3a2a-fe425958de32&l=262